src/letters/LetterPoints.js

```diff
@@ -0,0 +1,30 @@
+const letterPoints = [ 
+    { key: 1, letter: 'A', point: 1 }, 
+    { key: 2, letter: 'B', point: 4 },
+    { key: 3, letter: 'C', point: 5 },
+    { key: 4, letter: 'D', point: 3 },
+    { key: 5, letter: 'E', point: 2 },
+    { key: 6, letter: 'F', point: 5 },
+    { key: 7, letter: 'G', point: 4 },
+    { key: 8, letter: 'H', point: 4 },
+    { key: 9, letter: 'I', point: 3 },
+    { key: 10, letter: 'J', point: 7 },
+    { key: 11, letter: 'K', point: 5 },
+    { key: 12, letter: 'L', point: 4 },
+    { key: 13, letter: 'M', point: 3 },
+    { key: 14, letter: 'N', point: 3 },
+    { key: 15, letter: 'O', point: 2 },
+    { key: 16, letter: 'P', point: 3 },
+    { key: 17, letter: 'Q', point: 8 },
+    { key: 18, letter: 'R', point: 3 },
+    { key: 19, letter: 'S', point: 3 },
+    { key: 20, letter: 'T', point: 3 },
+    { key: 21, letter: 'U', point: 3 },
+    { key: 22, letter: 'V', point: 6 },
+    { key: 23, letter: 'W', point: 5 },
+    { key: 24, letter: 'X', point: 6 },
+    { key: 25, letter: 'Y', point: 4 },
+    { key: 26, letter: 'Z', point: 7 },
+  ];
+
+export { letterPoints };
```

Comment:
nit pick: this file is missing an ending newline

Reply:
Done